exchanges/request/request.go

```diff
@@ -455,6 +471,34 @@ func (r *Requester) SendPayload(method, path string, headers map[string]string,
 	return resp.Error
 }
 
+// GetNonce returns a nonce for requests. This locks and enforces concurrent
+// nonce FIFO on the buffered job channel
+func (r *Requester) GetNonce(isNano bool) int64 {
```

Comment:
What do you think about returning a nonce.Value instead, that way you can use the built in String() method so you don't have to worry about strconv.

Reply:
Good suggestion will change.